index.js

```diff
@@ -1107,6 +1108,9 @@ const _start = () => {
     if (url === '.') {
       console.warn('NOTE: You ran `exokit . <url>`\n(Did you mean to run `node . <url>` or `exokit <url>` instead?)')
     }
+    if (url && url.indexOf('://') < 0) {
```

Comment:
If the user passes in a url in a query string this is broken, isn't it?

Reply:
Yes, good catch.